filter/basiclti/post.php

```diff
@@ -132,7 +133,7 @@
 
 try {
 	$parms = signParameters($parms, $line['url'], "POST", $line['ltikey'], $line['secret'], null, $org_id, $org_desc);
-	$content = postLaunchHTML($parms, $line['url'],isset($parms['custom_debug']));
+	$content = postLaunchHTML($parms, Sanitize::rawurlencodePath($line['url']),isset($parms['custom_debug']));
```

Comment:
Check on this - I think this will cause problems.

Reply:
Change to encodeUrlForHref, and move encoding into postLaunchHTML function, to put encoding at point of output.